### PR TITLE
manage-and-coverage

### PR DESCRIPTION
**Pull request by scottf**
get consumers / consumer names
get streams / stream names
lots of coverage holes.

### CODE REVIEW

File: src/main/java/io/nats/client/impl/NatsJetStream.java [hunk 1]
```diff
@@ -307,57 +367,9 @@ private PublishAck publishInternal(Message message, PublishOptions options) thro
         return ack;
     }
 
-    private boolean isStreamSpecified(String streamName) {
```
Comment (scottf) [inline src/main/java/io/nats/client/impl/NatsJetStream.java:310]: Just moved this.

File: src/main/java/io/nats/client/impl/NatsServerInfo.java [hunk 2]
```diff
@@ -362,27 +362,4 @@ public String toString() {
                 ", cluster='" + cluster + '\'' +
                 '}';
     }
-
-    public String toJsonString() {
```
Comment (scottf) [inline src/main/java/io/nats/client/impl/NatsServerInfo.java:366]: Not used anywhere.

File: src/examples/java/io/nats/examples/ExampleUtils.java [hunk 3]
```diff
@@ -137,11 +135,12 @@ public static void sleep(long millis) {
     }
 
     public static void formatPrint(Object o) {
-        System.out.println(printable(o.toString()) + "\n");
+        // TODO
```
Comment (ColinSullivan1) [inline src/examples/java/io/nats/examples/ExampleUtils.java:138]: Still a TODO?

File: src/main/java/io/nats/client/impl/NatsJetStream.java [hunk 4]
```diff
@@ -155,12 +163,64 @@ public void deleteConsumer(String streamName, String consumer) throws IOExceptio
...
+    }
+
+    @Override
+    public List<String> getConsumerNames(String streamName, String filter) throws IOException, JetStreamApiException {
+        String subj = String.format(JSAPI_CONSUMER_NAMES, streamName);
+
+        ConsumerNamesResponse cnr = new ConsumerNamesResponse();
+        while (cnr.hasMore()) {
+            Message resp = makeRequestResponseRequired(subj, cnr.nextJson(filter).getBytes(StandardCharsets.US_ASCII), requestTimeout);
+            cnr.update(extractJsonThrowOnError(resp));
```
Comment (ColinSullivan1) [inline src/main/java/io/nats/client/impl/NatsJetStream.java:177]: Might want to rename this to `add()`?  Looks like it's not idempotent when applied to the same set.

**Review by ColinSullivan1 — APPROVED** [review]
LGTM.  One nitpick, up to you...  Thanks!

**Pull request merged**